backend/Earthfile

```diff
@@ -0,0 +1,16 @@
+VERSION 0.7
+FROM python:3.11
+WORKDIR /backend
+
+deps:
+    RUN pip install poetry
+    COPY pyproject.toml ./
+    COPY poetry.lock ./
+    RUN poetry build
+
```

Comment:
Missing stages:
docker (for docker builds)
check-format (for formatting)

Reply:
we didin't decide on a formatter yet tho